### ISSUE TITLE
[crash] Something went wrong

### ISSUE THREAD

**Issue by chargio** [kubeshop/monokle#2598]
**Describe the bug**

An error when selecting side by side on a file. (just starting the application). 

**Steps to reproduce**

Open Monokle-demo, and select in resources blue-cms without any other previous click
Select form side by side. See error

**Stacktrace** 

 ```
Cannot read properties of null (reading 'getValue')
 TypeError: Cannot read properties of null (reading 'getValue')
    at MonacoEditor.componentDidUpdate (file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:620106:64)
    at Yi (file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:613878:141)
    at dk (file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:615089:19)
    at exports.unstable_runWithPriority (file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:635065:12)
    at gg (file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:611355:10)
    at Uj (file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:614883:3)
    at Lj (file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:614530:3)
    at file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:611384:18
    at exports.unstable_runWithPriority (file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:635065:12)
    at gg (file:///Applications/Monokle.app/Contents/Resources/app.asar/build/bundle.main.js:611355:10)
```

**Comment by chargio** [comment]
It happens in every resource in the repo

**Comment by topliceanurazvan** [comment]
This error was handled in https://github.com/kubeshop/monokle/pull/2572. 

It will be fixed in `1.12.0`.
